bill/outlay.go

```diff
@@ -50,7 +50,7 @@ func (o *Outlay) Validate() error {
 // UnmarshalJSON helps migrate the desc field to description.
 func (o *Outlay) UnmarshalJSON(data []byte) error {
 	type Alias Outlay
-	aux := &struct {
+	aux := struct {
```

Comment:
Minor thing: this change should better be part of https://github.com/invopop/gobl/pull/234